spec/supabase_js_v2.yml

```diff
@@ -3701,93 +2294,8 @@ functions:
             insert into
               countries (id, name)
             values
-              (1, 'Afghanistan'),
-              (2, 'Albania'),
-              (3, 'Algeria');
-            ```
-          response: |
```

Comment:
looks like we cut too much here? @Hallidayo

Reply:
Looks okay on my editor so not sure if its showing in Github weirdly?

Reply:
👍 got it. no problem.

i'll get someone else to verify these changes since they are quite broad